suites/pacific/cephfs/tier-0_cephfs_upgrade_5x_to_5x.yaml

```diff
@@ -0,0 +1,175 @@
+#===============================================================================================
+# Cluster Configuration:
+#    conf/pacific/cpehfs/tier-0.yaml
+#
+# Test Steps:
+#   (1) Bootstrap cluster using latest released ceph 5.x with below options,
+#       - rhcs-version: 5.1 in this suite, but it can be changed to 5.x for upgrade start version as needed
+#       - release: ga in this suite but it can be changed to  <ga | z1 | z1-async1>
+#       - registry-url: <registry-URL>
+#       - mon-ip: <monitor address, Required>
+#   (2) Copy SSH keys to nodes and Add it to cluster with address and role labels attached to it.
+#   (3) Upgrade to distro build(N) by provided Ceph image version.
+#===============================================================================================
+---
+tests:
+  -
+    test:
+      abort-on-fail: true
+      desc: "Setup phase to deploy the required pre-requisites for running the tests."
+      module: install_prereq.py
+      name: "setup install pre-requisistes"
+  -
+    test:
+      abort-on-fail: true
+      config:
+        steps:
+          -
+            config:
+              args:
+                rhcs-version: 5.1
```

Comment:
Can we try modifying this parameter to 5.0,5.2 and 5.3 and validate if it brings up right builds using https://access.redhat.com/solutions/2045583

Reply:
@AmarnatReddy I can try that. Do you mean before upgrade version? or target upgrade version?

Reply:
I will valiadate 
- 5.0 -> 5.1, 5.2,5.3
- 5.1 -> 5.2-> 5.3
- 5.2 ->5.3